From e53723f6b83740a68266700fd89c91dc0cb1978e Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Wed, 2 Sep 2020 11:17:37 +0200
Subject: [PATCH 352/529] dt-bindings: remoteproc: add compatibility for TEE
 support

Rework compatibility description according to the support of
the authenticated firmware relying on OP-TEE authentication

Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: I890e78d98fd097463ea6b85a4b05b320732f90dd
---
 .../devicetree/bindings/remoteproc/st,stm32-rproc.yaml        | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

--- a/Documentation/devicetree/bindings/remoteproc/st,stm32-rproc.yaml
+++ b/Documentation/devicetree/bindings/remoteproc/st,stm32-rproc.yaml
@@ -16,7 +16,9 @@ maintainers:
 
 properties:
   compatible:
-    const: st,stm32mp1-m4
+    enum:
+      - st,stm32mp1-m4
+      - st,stm32mp1-m4_optee
 
   reg:
     description:
